<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigWidget</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelIdFile">
     <property name="text">
      <string>ID file:</string>
     </property>
     <property name="buddy">
      <cstring>idFile</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="idFile">
     <property name="whatsThis">
      <string>Name of ID file, as generated by mkid(1).</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelKeySize">
     <property name="text">
      <string>Complete tokens after:</string>
     </property>
     <property name="buddy">
      <cstring>keySize</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KIntSpinBox" name="keySize">
     <property name="whatsThis">
      <string>Minimum length of token before completions will be shown.</string>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Lookup in files</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSrcIn">
        <property name="text">
         <string>Discard file prefix ending with key:</string>
        </property>
        <property name="buddy">
         <cstring>idFile</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="srcIn">
        <property name="whatsThis">
         <string>Discard the first part of the file name ending with this key.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSrcOut">
        <property name="text">
         <string>Then add file prefix:</string>
        </property>
        <property name="buddy">
         <cstring>srcOut</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="srcOut">
        <property name="whatsThis">
         <string>Replacement prefix. Use %{idPrefix} to insert the prefix of the ID file ending with the key.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelUseSuffixes">
        <property name="text">
         <string>Highlight files with suffixes:</string>
        </property>
        <property name="buddy">
         <cstring>useSuffixes</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KLineEdit" name="useSuffixes">
        <property name="whatsThis">
         <string>Use .h;.hxx to highlight files which may contain declarations.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="useEtags">
        <property name="whatsThis">
         <string>Use etags(1) to find definitions, and highlight them.</string>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelUseEtags">
        <property name="text">
         <string>Highlight definitions using etags(1):</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>idFile</tabstop>
  <tabstop>keySize</tabstop>
  <tabstop>srcIn</tabstop>
  <tabstop>srcOut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
